after/config.py

```diff
@@ -20,10 +25,3 @@ class Params:
     epoch_count: int
     lr: float
     batch_size: int
-
-
-@dataclass
-class MNISTConfig:
```

Comment:
I could have not removed this dataclass. I did it to have multiple parameters in the `main` function.

Instead of having a `config.py` module, with jsonargparse I normally define the parameters in the functions and class methods where they are used. A dataclass in a separate module only if it is a group of settings that are used in multiple places.